SPB/Platform/EGL/EGLHelper.cs

```diff
@@ -0,0 +1,159 @@
+using System;
```

Comment:
naming is all over the place here + the formatting is wrong for bracket in some places